bahatii/Property-Pro-lite
# finishes #167097830 view_details

#### What does this PR do?
enable the user to view the specific property in details
#### Description of Task to be completed?
the zoomed picture of the products
the side details of the products
view similar products
#### How should this be manually tested?
click on the first property on the view- products-user page
it should show the details of the first product
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#167097830
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A
